Runtime/CGameHintInfo.cpp

```diff
@@ -31,12 +31,12 @@ CGameHintInfo::SHintLocation::SHintLocation(CInputStream& in, s32)
 , x8_areaId(in.readUint32Big())
 , xc_stringId(in.readUint32Big()) {}
 
-int CGameHintInfo::FindHintIndex(const char* str) {
+int CGameHintInfo::FindHintIndex(std::string_view str) {
```

Comment:
const? or does that not matter with string_view

Reply:
Doesn't matter with string_view